### ISSUE TITLE
`open` opens a new instance of atom rather than existing instance

### ISSUE THREAD

**Issue by AgentAntelope** [atom/atom#8437]
This is inconsistent with standard app behaviour.

OS X 10.10.4, Atom 1.0.7

**Comment by batjko** [comment]
You mean `Open Folder...`, I assume?

As far as I know, the default behavior should be to open the folder into the same instance of Atom if that instance doesn't already have a folder open. Otherwise, as you mention, it opens it in a new window instead.

This has been the subject of a lot of debate since Atom was open-sourced and this behavior was what most people could agree on.

**Comment by mnquintana** [comment]
@AgentAntelope Do you mean that Atom opens a second instance when opening from the command line? If so, this sounds like the same issue as https://github.com/atom/atom/issues/2964.

**Comment by AgentAntelope** [comment]
@mnquintana that's partially the issue; However I am also seeing atom open a blank window (untitled tab, no sidebar, etc) even when existing project windows are open.

**Comment by AgentAntelope** [comment]
To expand on this: If I have atom open, running `open /Applications/Atom.app` causes this behaviour.

**Comment by AgentAntelope** [comment]
This is a pretty low quality gif, but it shows the issue ![atom_issue_8437](https://cloud.githubusercontent.com/assets/1962412/9356161/ba7fc886-4676-11e5-82d6-8db12efc7c2f.gif)

**Comment by batjko** [comment]
Oh, you're talking Mac's "open" - and opening two instances of Atom. 

I just tried it on Linux and the second time I open Atom (without passing any parameters) it did *not* open a second instance for me. Ubuntu just (literally) shook the existing open App icon, telling me it's already open.

**Comment by mnquintana** [comment]
Hmm, I just tried it out with Atom 1.0.7 and OS X 10.10.4, and when I ran `open /Applications/Atom.app` it just switched focus to the already running instance of Atom.

Can anyone else reproduce this? @atom/feedback

**Comment by AgentAntelope** [comment]
Potentially relevant information: I am installing atom with brew-cask. However, opening the symlink's target has the same effect. (`/opt/homebrew-cask/Caskroom/atom/1.0.7/Atom.app` being the resolution of that link)

**Comment by AgentAntelope** [comment]
I am no longer able to replicate this issue.

**Comment by maxbrunsfeld** [comment]
@AgentAntelope Ok, I'm going to close this out. Let me know if you see this in the future.

**Comment by AgentAntelope** [comment]
After some investigation: the issue was caused by updating atom via brew-cask whilst it was running. This causes the application opened via the symlink to be a different instance than the running one, opening the new install when running `open`.

TL;DR: it's my fault, this issue should stay closed.
